datasets/usgs-landsat.yaml

```diff
@@ -0,0 +1,119 @@
+Name: USGS Landsat
+Description: |
+  This joint NASA/USGS program provides the longest continuous space-based record of 
+  Earth’s land in existence. Every day, Landsat satellites provide essential information 
+  to help land managers and policy makers make wise decisions about our resources and our environment.
+  Data is provided for Landsats 4, 5, 7, and 8.
```

Comment:
Level 2 data is only provided for Landsat 4-8, but there's also level 1 data going back to Landsat 1 in the bucket too it looks like.
```
> aws s3 cp s3://usgs-landsat/collection02/level-1/standard/mss/1972/200/050/LM01_L1TP_200050_19720928_20200909_02_T2/LM01_L1TP_200050_19720928_20200909_02_T2_stac.json - --request-payer | jq '.properties."eo:platform"'
"LANDSAT_1"
```

Reply:
Good catch! Hadn't scrolled through the STAC records enough to see that.